main.py

```diff
@@ -16,12 +16,33 @@
 import datetime, os, uuid
 
 from flask import Flask, render_template, redirect, url_for, request, abort, Response, jsonify
+from flask_mail import Mail, Message
 from db_operations import *
 from forms import AdoptionForm, CreateListingForm
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = os.urandom(32)
+app.config['MAIL_SERVER'] = 'smtp.googlemail.com'
+app.config['MAIL_PORT'] = 587
+app.config['MAIL_USE_TLS'] = True
+app.config['MAIL_USERNAME'] = 'petadoption.sps@gmail.com'
+app.config['MAIL_PASSWORD'] = os.getenv("email_pwd") # SET password as environment variable
```

Comment:
I think it would be better if this environment variable was also `MAIL_PASSWORD`